### DISCUSSION TITLE
Development plans and timetables

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#26] [Planning consideration]
# Development plans and timetables

Development plans, such as local plans, set out the priorities and policies for how an area should develop and grow.

## What

We are going to look at data that will help with finding plans and data that records what happens during the plan-making stage.

## Why (we are looking at it)

In DLUHC we often need to look at local plans and the first step of that is to know where to find the right one for a Local Planning Authority.
It can be difficult to get good, up-to-date data on where in the plan-making process a plan is. This can have consequences for citizens hoping to engage with the process, for policymakers hoping to assess how the policy is working and for the planning inspectorate which needs to be able to plan when they can examine a plan.

## What we're looking for

We want to know more about:

* who needs development plan data
* what they want to use it for
* if anyone asking for this data/information
* what should be included in the data
* the stages in the plan making process

**Comment by colmjude** [comment]
We have drafted [a standard for development plans](https://digital-land.github.io/specification/specification/development-plan/).

It includes schemas to capture a record of plans (existing and emerging), a timeline of events related to the plan-making process and associated documents publishers believe should be recorded (for example, the official adoption notices).

The standard includes numerous "category" fields, for example, `development-plan-event`. More research and analysis of existing data is needed before we can decide on what values will be part of each category field.

**Comment by colmjude** [comment]
We recently updated the development-plan specification we originally drafted in April 2023.

The most significant change is introducing the development-plan-geography dataset. During our testing, we realised some plans cover areas that do not match up nicely with the boundary of the authoring local planning authority.

For example,

* [Shoreham harbour joint area action plan](https://development-plan-prototype.herokuapp.com/development-plan/shoreham-harbour-joint-area-action-plan) - this plan covers bits of 3 LPA districts
* [Chichester Local plan](https://www.chichester.gov.uk/newlocalplan) - this plan covers the Chichester local authority district excluding the National Park area
* [Tower Hamlets policies map](https://www.towerhamlets.gov.uk/Documents/Planning-and-building-control/Strategic-Planning/Local-Plan/Policies-Map-15-01-2020.pdf) - this plan covers the Tower Hamlets area minus an area covered by The London Legacy Development Corporation.

Due to this, it makes sense to provide an exact geometry for the area any plan covers. We’ve created a separate dataset rather than including a `geometry` field in development-plan records. This allows us to categorise the geographies with a development-plan-geography-type. We have seen 3 different geography types: local-authority-district, joint-plan-geography and bespoke areas. The names of the 3 types are still under consideration.

We also updated the `organisation` field to `organisations` so that recording a jointly written plan is easier.

Our final change is introducing an `adopted-date` field to development-plan records.

The point at which a plan is “adopted” is the significant point in the lifecycle of a plan. It is the point at which the legal standing of a plan materially changes, and it makes sense to record that at the plan record level.

Doing so means someone using only the development-plan dataset can determine which plans are adopted and which are not, without having to search through the development-plan-timetable dataset.

Taking this approach, where we include a meaningful date on a top-level record is an emerging pattern. We take a similar approach with [tree-preservation-orders](https://digital-land.github.io/specification/specification/tree-preservation-order/) and [planning-applications](https://digital-land.github.io/specification/specification/planning-application/).

**Reply by colmjude** [reply]
I should add that when we discuss the 3 geograhy types: local-authority-district, joint-plan-geography and bespoke areas. We are not proposing those as the names. They are categorisations we are using for now whilst we work out more suitable names/labels for them.
Any suggestions welcome!

**Reply by colmjude** [reply]
Here is the latest entity diagram which includes the `development-plan-geography` dataset

![development-plan-with-geography-with-type](https://github.com/digital-land/data-standards-backlog/assets/299875/9f074d4f-fb41-42b5-94e0-52bace98ae2d)

**Reply by colmjude** [reply]
We are planning to rename the `development-plan-geography` dataset to `development-plan-boundary`. Based on feedback we feel this name is clearer.

To make this change we will need to 

* rename `development-plan-geography` to `development-plan-boundary`
* rename `development-plan-geography-type` dataset to `development-plan-boundary-type`

**Reply by colmjude** [reply]
We've made these changes now. Commits

* [6bda3d7](https://github.com/digital-land/specification/commit/6bda3d7)
* [7875612](https://github.com/digital-land/specification/commit/7875612)

**Reply by psd** [reply]
The field `development-plan-boundary-type` isn't a good name and worth reviewing as there is a potential for a more general pattern here for how we link to another boundary this entity has been derived from. 
 
One potential idea would be to set the prefix to be `statistical-geography` and the reference the LPA GSS code where it has been copied in, and `local-plan-boundary ` with a reference created for this dataset where it has been created for this joint plan.

Also we can configure the pipeline to run LPA resources for the `local-plan-boundary` dataset, using lookups to assign entities to the boundaries. It's possible for a local plan area not to have a local plan, and the map will show where where they are covered by a joint local plan boundary. 
 
So I think we should remove the `-boundary-type` field for now.

**Reply by colmjude** [reply]
Thanks @psd 
Yes agreed. We included it when we wanted to differentiate between boundaries that are exact matches to LPA boundaries, joint boundaries and boundaries that are specific to the local plan (for example, Tower Hamlets) but agreed it is not an overly helpful field so we'll remove it.

> the reference the LPA GSS code where it has been copied in

Yes, absolutely, this is exactly what we've been doing when seeding the local plan data. Here is the [local-plan-boundary dataset](https://github.com/digital-land/local-plans-explorer/blob/main/data/export/local-plan-boundary.csv) of the boundaries that have so far been review and made available for the platform. All the `references` are the statistical-geography codes/references.

Extract from csv

| reference | name | organisations | geometry | ... |
| --- | --- | --- | --- | --- |
| 2024-10-14 | E09000007 | London Borough of Camden | local-authority:CMD | ... |
| 2024-10-14 | E09000019 | ... |  ... | ... |
| ... | ... | ... | ... | ... |

Thats an interesting idea.

Just to check I understand correctly, are you suggesting that even if we have a joint plan, such as **Babergh and Mid Suffolk join local plan**, a user would be able to click on the `local-plan-area` for Babergh and it'd go through to the Babergh and Mid Suffolk plan?

**Reply by colmjude** [reply]
Removed `x-boundary-type` field from local-plan and development-plan specifications.

Commits:

* [d9010e8](https://github.com/digital-land/specification/commit/d9010e80bf1d42e24ab3ea1f7afd34f56e79dbd9)
* [82bbda8](https://github.com/digital-land/specification/commit/82bbda8)

**Comment by mattlucht** [comment]
In a conversation with the DLUHC local plans team a couple of questions came up around how we might handle certain scenarios of the local plan timetable:

* in the case of a joint plan there may be a scenario where the adopted date of the plan differs between local planning authorities, there is a question as to whether we keep `adopted-date` in the `development-plan` dataset or can it be inferred from beng an event in the `development-plan-timetable`?
* it is anticipated that each local planning authority will need to publish their own timetable. The data model supports adding multiple `organisations` to the `development-plan` (in the case of a joint plan), but the local plans team are considering how they handle this scenario in the tool(s) that they're building. At the moment the proposed tool to help create timetables won't hold any data but in the case of a joint plan each local planning authority will need to link their `development-plan-timetable` events back to a specifc `development-plan`

**Comment by colmjude** [comment]
## Events for the timetable

The key events we have for the local plan making process are

| name | reference | description |
| --- | --- | --- |
| Plan adopted | plan-adopted | |
| Inspector report published | inspector-report-published | |
| Examination hearing start | examination-hearing-start | |
| Examination hearing end | examination-hearing-end | |
| Submit plan for examination | submit-plan-for-examination | |
| Draft plan for public consultation published | draft-plan-for-public-consultation-published | |
| Public consultation start | public-consultation-start | |
| Public consultation end | public-consultation-end | |
| Planning inspectorate examination start | planning-inspectorate-examination-start | |
| Planning inspectorate examination end | planning-inspectorate-examination-end | |
| Planning inspectorate found sound | planning-inspectorate-found-sound | |
| Timetable published | timetable-published | |

We want to keep this list as short and meaningful as possible. The more options the harder it becomes to maintain the data so ideal each event will have a strong need.

If you think we are missing any key events you thing should be log and that you'd find useful please let us know.

**Reply by colmjude** [reply]
When developing local plans Planning Authorities produce a Local Development Scheme that forecasts when they think things will be done or completed.

Originally we thought we'd be able to use the same events to record this. Essentially, if the `event-date` was in the future then it'd be a forecast - a planning authority saying we expect event X to happen on this date.

Then they could update the `event-date` when it actually happened.

However, we don't think this will work. If a provider forecasts a date and then never updates the record then someone using the data would, rightly, assume the event occurred on that date. This is a potential problem.

To avoid this we are going to include events that are specifically for forecasting. So for the event around public consultation start date we'd have:
* `expected-public-consultation-start` - used to predict when this will happen
* `public-consultation-start` - when the event happened

This way there will be less chance of misinterpreting the data. And, users would be able to compare the actual date with the forecast date.

**Reply by colmjude** [reply]
### Updating the events

We’ve reworked the development plan events to better reflect the process for creating a local plan. In particular, the events around the consultations.

Previously, we had `public-consultation-start` and `public-consultation-end` events, which we thought would work for any consultations. For example, if 2 consultations happened, we could have 2 rows, one for each. For example

| event | event-date | notes |
| --- | --- | --- |
| public-consultation-start | 2023-07-01 | The start of the reg 18 consultation |
| public-consultation-end | 2023-08-15 | The end of the reg 18 consultation |
| public-consultation-start | 2023-11-01 | The start of the reg 19 consultation |
| public-consultation-end | 2023-12-15 | The end of the reg 19 consultation |

However, now we understand that there are 2 different consultations that serve different purposes. So we have introduced events for both so that for users of the data it is clear what type of consultation was or will be done at a given time. It will no longer rely on optional explainer text in the `notes` field.

| event | event-date | notes |
| --- | --- | --- |
| reg-18-public-consultation-start | 2023-07-01 |  |
| reg-18-public-consultation-end | 2023-08-15 |  |
| reg-19-public-consultation-start | 2023-11-01 |  |
| reg-19-public-consultation-end | 2023-12-15 |  |

We manage these changes in our dataset editor. For the[ development-plan-events](https://dluhc-datasets.planning-data.dev/dataset/development-plan-event), we’ve added end-dates to old events and added new events.

We have also started work on a [“finder” page (this is a first draft)](https://dluhc-datasets.planning-data.dev/dataset/development-plan-event/finder). This will help data providers find the specific value to use for the event they want to reference in their data.

**Comment by psd** [comment]
I noticed we renamed `development-plan-geography` dataset to `development-plan-boundary` but not updated the `development-plan` dataset field, so have updated the spec to reflect this change: https://digital-land.github.io/specification/specification/development-plan/

**Comment by nikkiwebber** [comment]
Hi all - I discussed this with @stevenjmesser today. The CoLC are putting together some data sets to these specifications and I wanted to share some feedback from our policy team.  Questions are as follows 1) we're looking to add the London Plan into our development-plan-document-list a) should this be something we do, or is this for the GLA to do? and b) of the document-types listed in the specification, none fit the London Plan (they have suggested "spatial-development-strategy" and fed back that other LPAs with a mayoral authority might offer the same feedback. 2) re the development plan, the development plan comprises (amongst other things) our local plan, and the London Plan at the time. Should we be listing the London Plan as two records in the development-plan-document list, once for when it applied in with our previous Local Plan, and once for when a new Local Plan is adopted? Happy to chat through this in detail if that helps.

**Reply by psd** [reply]
Hi Nikki. The London Plan should be provided by the GLA, we can add it for them, but the GLA are the authoritative source "spatial-development-strategy" sounds like a document-type we can add.

**Reply by stevenjmesser** [reply]
@colmjude, do you know the answer to Nikki's second question?

**Comment by nikkiwebber** [comment]
Hi All - We have finalised our data sets for 
•	development-plan-boundary
•	development-plan
•	development-plan-document
•	development-plan-timetable
•	local-plan-boundary
•	local-plan
•	local-plan-document
•	local-plan-timetable

My planning officers have kindly provided some feedback - which we'd be happy to talk through in more detail. Please let me know if I should add this to the github discussion on Local Plans too. 

Feedback is below for you. 
-	local-plan : We have currently recorded three versions of our local plan, our Local Plan 2015, which was adopted. The City Plan has two versions, one that took us up to 2036, and then a revised plan called the City Plan 2040. Technically they are evolutions of the same document, and cover different periods, however, elements of the timetable apply to both plans. I've currently handled this in the data set by providing an array for the local-plan in the local-plan-timetable against the relevant event. 
-	development-plan-timetable : We have not added timetable events for documents which are created by other organisations (i.e. the London Plan) – although these events should be included in our development-plan-timetable. We assume over time this data will be gathered from other organisations, and pulled through into our data set, by virtue of us having used their reference (eg the GLA London Plan). 
-	development-plan-document : Not all the London Plan documents were available for us to be entirely accurate with the records of our Development Plan – in order to be accurate we would require there to be records for the previous 2 versions of the London Plan. 
-	development-plan-document : Similarly there is no record for the 2018 Local Aggregate Assessment for London and the South-East London Joint Waste Planning Group Technical Report, and so we are unable to include the relevant Minerals and Waste Plan in our development-plan-documents data set. 
-	local-plan-document : We think there would be a benefit to being able to include evidence base documents in the local-plan-document data set. We believe the easiest way to do this is to include a boolean field in the local-plan-document data set, to indicate which of the records make up the dataset, as well as including evidence-base-document as a document-type. We have reasons as to why it wouldn’t be appropriate to only include the evidence-base-document as a document-type to select, which we’d be happy to talk through with you. 
-	local-plan-document and development-plan-document : Supplementary planning documents are not part of any of these datasets, and therefore shouldn’t be part of the available document-type
-	The Local Plans explorer tool for the Local Plan boundary generates a data set for the organisation of the City of London; this is incorrect, it should generate the boundary for the City of London Corporation as Planning Authority (statistical-geography:E60000189)
-	development-plan : We have taken the view of recording only instances of the development plan as it currently exists, and events that are associated with that development plan. We appreciate there is an adopted-date data field, so we’d appreciate some guidance on whether we should be including emerging, but not adopted or legally relevant, versions of the development plan, with the adopted-date left blank if it is not in force? If the latter then we will update this data set accordingly. 
-	local-plan : We feel that there should be an option to record draft versions of the same local plan – or different versions that would be different at various points of publication, but technically are the same plan. This may have been the intention with the local-plan data set, but we couldn’t see a way to make it obvious that there could be multiple versions of the same plan. It would be useful to be able to be able to have multiple versions of the same plan, so we can be specific about which version was consulted on in the “events” tab, as well as be able to connect documents and potentially consultation comments to the correct version of the plan as it was at a specific time. We’d welcome the opportunity to engage with you on this.

**Comment by psd** [comment]
I think this consideration is a broader collection of individual considerations, including "local plans", "neighbourhood plans", "minerals plan", "waste plan" in the same way our "organisation" dataset is a combination of "local-authority", "national-park-authority" etc
